### ISSUE TITLE
Regarding test data detections used in the repo.

### ISSUE THREAD

**Issue by Umar1998** [damo-cv/MMP_Track1_ICCV21#3]
Hi, I am doing Research for my Master's thesis on multi camera multi object tracking. I have seen your presentation, that was really nice. I was going through your code. You have provided the detections for the test data.  How did you got the test data.

I have mailed the ICCV2021 team, and I got the dataset link for train and validation set but not the test set that you used. 

I am actually hoping that whether you can provide some link to the test data or some refence for that so that I can understand your code in a better way. This would help a lot in my thesis project.

Thanks.
